### PR TITLE
Portfolio: Form validation and submit

### PR DESCRIPTION
**Pull request by usmansbk**
In this activity, I:

- Added form validation
- Added form submit URL

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -301,20 +324,42 @@ <h3>
...
-        <input type="email" id="useremail" name="useremail" placeholder="enter your email" value="natajayan14@gmail.com">
-        <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"></textarea>
+      <form action="https://formspree.io/f/xknkbwnn" method="POST">
+        <input
+          type="text"
+          id="username"
+          name="username"
+          placeholder="Enter your name"
+          required
+          minlength="30"
```
Comment (Dipeshtwis) [inline index.html]: - [ ] Message input has `maxlength="30"` attribute not `minlength` as you know name can be of four letter, five letter, that's why we have to use max to restrict the username length to 30.

File: index.html [hunk 2]
```diff
@@ -301,20 +324,42 @@ <h3>
...
+          required
+        />
+        <textarea
+          name="message"
+          id="message"
+          cols="30"
+          rows="10"
+          placeholder="Write your message here"
+          required
+          minlength="500"
```
Comment (Dipeshtwis) [inline index.html]: - [ ] Message input has `maxlength="500"` attribute not `minlength` as you know not everyone willing to write above 500 letter, so we restrict the input 500 to make the message short and to the point.

File: index.html [hunk 3]
```diff
@@ -301,20 +324,42 @@ <h3>
...
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
         </p>
       </div>
-      <form action="/contact" method="POST">
-        <input type="text" id="username" name="username" Placeholder="Enter your name" value="Yeremias">
-        <input type="email" id="useremail" name="useremail" placeholder="enter your email" value="natajayan14@gmail.com">
-        <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"></textarea>
+      <form action="https://formspree.io/f/xknkbwnn" method="POST">
```
Comment (Dipeshtwis) [inline index.html]: - [ ] All the inputs are overflowing and hidden from the left on the screen. Please make sure to give proper styling to fix this and make it look like figma design.

your design:

Screenshot from 2021-06-08 17-35-50

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Sly-Cloud, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Your commits are looking good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] Update the live link with the current design in README file.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Sly-Cloud) [reply index.html]: @Dipeshtwis Thanks for the review, I have made most of these changes, however, please I'd appreciate it if you check again, I see the color difference between this screenshot you sent, and what I actually have on the repo.

Please check again

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @Sly-Cloud 

##### Great work on making some of the  changes requested by the previous reviewer:clap::clap::clap:
You've done a good job implementing some of the requested changes:blush:
but there are still some which aren't done yet, 
so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [ ] Please check the original design below and compare it with yours.
- [ ] kindly fix the the styling and make good use of flex box
- [ ] Please do not merge until the project is approved 

## Original design 
right

## your project now

wrong



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by Sly-Cloud** [comment]
Thanks @Shaher-11 for the review, I was able to see and rectify this. Please help me confirm it with the live demo. Awaiting you response 😃

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉 🎉

Hi @Sly-Cloud , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**